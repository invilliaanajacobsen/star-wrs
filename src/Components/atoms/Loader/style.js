import styled, { keyframes } from 'styled-components';
import Row from 'react-bootstrap/Row'


export const Line = styled(Row)`
	display: flex;
	align-items: center;
	justify-content: center;
`;


export const spin = keyframes`
from { transform: rotate(360deg); }
to { transform: rotate(0deg); }
`

export const Loading = styled.div`
	border: 10px solid #f3f3f3;
	border-radius: 50%;
	border-top: 10px solid #343a40;
	width: 60px;
	height: 60px;
	animation: ${spin} 1s linear infinite;
	margin-top: 5%;
`;

