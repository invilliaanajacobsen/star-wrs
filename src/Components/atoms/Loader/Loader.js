import React from 'react';
import { Loading, Line } from './style.js';

const Loader = props => (
	<Line>
		<Loading />
	</Line>
);

export default Loader;