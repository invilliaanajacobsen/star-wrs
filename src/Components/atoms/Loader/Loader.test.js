import React from "react";
import { shallow } from "enzyme";

import Loader from "./Loader";

describe("Loader component", () => {

	it("should render Loader match snapshot", () => {
		expect(shallow(<Loader />)).toMatchSnapshot();
	});
});
