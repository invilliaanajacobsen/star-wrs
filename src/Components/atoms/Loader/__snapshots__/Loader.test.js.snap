// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component should render Loader match snapshot 1`] = `
<Styled(Component)>
  <styled.div />
</Styled(Component)>
`;
